src/modules/rep.ts

```diff
@@ -181,19 +181,19 @@ export class RepModule extends Module {
 					rg.amount
 				} ${emoji}** on [message](${messageLink}) (<@${
 					rg.initialGiver
-				}>${rg.amount > 1 ? ' et al.' : ''}) at <t:${
-					(+new Date(rg.date) / 1000) | 0
-				}>`;
+				}>${rg.amount > 1 ? ' et al.' : ''}) at <t:${Math.floor(
+					+new Date(rg.date) / 1000,
+				)}>`;
```

Comment:
*Fiiine*

Reply:
_What if the bot is in use after year 2106?_